content/news/translation/2023/do-you-speak-prestashop-september-2023.md

```diff
@@ -0,0 +1,122 @@
+Here are the most active translators and proofreaders in September 2023.
+ 
+#### Top 10 translators in September 2023:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | Carlos Prince (CarlosPrince) | Spanish, Venezuela | 4,421
+| 2. | Cha (cafetango) | Chinese Traditional | 4,107
+| 3. | Rostyslav (config123) | Ukrainian | 3,235
+| 4. | Tatu Wikman (tswfi) | Finnish | 1,102
+| 5. | Rudolf Kastl (che666) | German | 1,091
+| 6. | Daniel Hlavacek (Hlavtox) | Czech; Slovak | 824
+| 7. | SNor42 | Hungarian | 707
+| 8. | Enzo Ferrara (ferribbotte) | italian | 704
+| 8. | AmirRzd | Persian | 704
+| 8. | David Gonzalez (davidglezz) | Spanish | 704
+| 8. | Ramon J. Batlle (rjbatllet) | Catalan | 704
+| 9. | Adeko Webdesign & Development (Adeko) | Dutch | 688
+| 9. | 한연일 (Huckhan) | Korean | 688
+| 10. | SeongHyeon Cho (jaymz9634) | Korean | 406
+
+
+#### Top 10 proofreaders in September 2023:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | Cha (cafetango) | Chinese Traditional | 4,088
+| 2. | Rostyslav (config123) | Ukrainian | 3,917
+| 3. | Rauno Riikman (weaver) | Finnish | 1,211
+| 4. | Daniel Hlavacek (Hlavtox) | Czech; Slovak | 749
+| 5. | Enzo Ferrara (ferribbotte) | Italian | 744
+| 6. | SNor42 | Hungarian | 707
+| 7. | AmirRzd | Persian | 704
+| 7. | David Gonzalez (davidglezz) | Spanish | 704
+| 7. | Ramon J. Batlle (rjbatllet) | Catalan | 704
+| 8. | Adeko Webdesign & Development (Adeko) | Dutch | 688
+| 9. | Angela Anna (Annajeronima) | Mongolian | 666
+| 10. | Stamatis (breezer)) | Greek | 409
+
+
+Congratulations, and welcome to the project's new top contributors. :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of September, PrestaShop 8 was fully available (= 100% translated and validated) in **11 languages**. That’s great! Thank you for your contribution.
+
+| **Catalan** | **Chinese Traditional** | **Czech** | **Dutch** | 
+| **English** | French | **Hungarian**| Italian | 
+| **Persian** | **Spanish** | **Ukrainian** | 
+ 
+#### Almost there!
+
+**18** languages are almost fully available: 
+
+| Greek | Korean | Portuguese | Slovak |
+
+There are less than 2% of strings to translate and proofread! You can do it! :muscle: 
+
+
+Congratulations and a huge thank you to all of you, contributors! :party-popper: 
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active.
+Indeed, some editing and rewriting may be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```

Comment:
```suggestion
Indeed, some editing and rewriting may be going on, but the translation percentage would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```